spring-cloud-stream-samples/pom.xml

```diff
@@ -21,6 +21,7 @@
 		<module>tap</module>
 		<module>double</module>
 		<module>extended</module>
+		<module>rxjava</module>
```

Comment:
Sure, but the XD sample reuses the general-purpose RxJava support from Spring XD - and I was suggesting something similar (with extracting the common behaviour as a reusable library - we surely don't want every new implementation to copy everything). Maybe SCSM is not the best location for this common library and we should have a `spring-cloud-stream-rxjava` project?